src/xforms/xformCartCheckout.js

```diff
@@ -60,7 +60,7 @@ function xformCartFulfillmentGroup(fulfillmentGroup, cart) {
     },
     // For now, we only ever set one fulfillment group, so it has all of the items.
     // Revisit when the UI supports breaking into multiple groups.
-    items: cart.items || [],
+    items: cart.items.filter(({ _id }) => fulfillmentGroup.itemIds.includes(_id)),
```

Comment:
Only include items from the current fulfillment group. Needed once you have more than on fulfillment group